### ISSUE TITLE
 Host profile window and internallauncher change to allow easier addition of launchers.

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#2134]
To add a new launch method for VisIt, one must add an option with the launch string (e.g. "llsubmit/mpirun") in the Host Profiles window and then change the internallauncher script to add a new job submitter class. It would be better to change how things work a little to avoid the step that requires the gui to be modified/recompiled.

1. Enhance internallauncher a little to enumerate the supported launcher/sublauncher configurations and store them in a VISIT_LAUNCHERS environment variable before starting the GUI. This should be fairly simple. Create all the job submitters and ask them what their launcher/sublaunchers are and add to a list which then makes the environment variable.
2. Change the Host profile GUI window so it populates the available launcher options based on the VISIT_LAUNCHERS environment variable. Give the user the option to type in a custom string for the launcher.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Brad Whitlock
Original creation: 02/17/2015 03:31 pm
Original update: 02/19/2015 08:00 pm
Ticket number: 2153
